dashboard/src/services/passwordReset.service.ts

```diff
@@ -0,0 +1,103 @@
+'server-only';
+
+import * as crypto from 'crypto';
+import { ForgotPasswordData, ResetPasswordData } from '@/entities/passwordReset';
+import { findUserByEmail, updateUserPassword } from '@/repositories/postgres/user';
+import {
+  createPasswordResetToken,
+  findPasswordResetToken,
+  deletePasswordResetToken,
+  deleteUserPasswordResetTokens,
+} from '@/repositories/postgres/passwordReset';
+import { sendResetPasswordEmail } from '@/services/email/mail.service';
+
+const TOKEN_EXPIRY_HOURS = 1;
```

Comment:
Quibble: Use an optional environment variable for this, with a default of 1 hour for instance. Thoughts?